src/ray/core_worker/transport/normal_task_submitter.cc

```diff
@@ -552,9 +552,9 @@ void NormalTaskSubmitter::PushNormalTask(
     const SchedulingKey &scheduling_key,
     TaskSpecification task_spec,
     const google::protobuf::RepeatedPtrField<rpc::ResourceMapEntry> &assigned_resources) {
-  RAY_LOG(DEBUG) << "Pushing task " << task_spec.TaskId() << " to worker "
-                 << WorkerID::FromBinary(addr.worker_id()) << " of raylet "
-                 << NodeID::FromBinary(addr.raylet_id());
+  RAY_LOG(INFO) << "Pushing task " << task_spec.TaskId() << " to worker "
```

Comment:
This might be too verbose and we have task events that cover the same thing?

Reply:
This log is generated only once for each normal task, which is totally acceptable in my opinion. In addition, I don't think events can replace logs. Events are just a supplement to help us quickly gain high-level insights into which components might be related to the root cause, but we still need logs for detailed debugging.

Reply:
It doesn't hurt the UX and makes our lives much easier.